controllers/datadogagent/agent_test.go

```diff
@@ -825,6 +825,20 @@ func defaultPodSpec() corev1.PodSpec {
 }
 
 func defaultSystemProbePodSpec() corev1.PodSpec {
+	agentWithSystemProbeVolumeMounts := []corev1.VolumeMount{}
+	agentWithSystemProbeVolumeMounts = append(agentWithSystemProbeVolumeMounts, defaultMountVolume()...)
+	agentWithSystemProbeVolumeMounts = append(agentWithSystemProbeVolumeMounts, []corev1.VolumeMount{
+		{
+			Name:      "sysprobe-socket-dir",
+			ReadOnly:  true,
+			MountPath: "/opt/datadog-agent/run",
+		},
+		{
+			Name:      "system-probe-config",
+			MountPath: "/etc/datadog-agent/system-probe.yaml",
+			SubPath:   "system-probe.yaml",
+		},
```

Comment:
Can the `system-probe-config` volume be mounted as read-only ?

```suggestion
		{
			Name:      "system-probe-config",
			MountPath: "/etc/datadog-agent/system-probe.yaml",
			SubPath:   "system-probe.yaml",
			ReadOnly:  true,
		},
```